src/openrct2/paint/support/Generic.h

```diff
@@ -0,0 +1,15 @@
+/*****************************************************************************
```

Comment:
`Supports.h` seemed like a better name, I guess

Reply:
I want to clean that up later anyway - I think it is just coincidentally the same value. Just didn’t want to change too much in one go.